modules/backup_replication/main.tf

```diff
@@ -0,0 +1,102 @@
+data "aws_caller_identity" "current" {}
+data "aws_elb_service_account" "elb_account_id" {}
+
+data "aws_region" "current" {}
+
+locals {
+  account_id = data.aws_caller_identity.current.account_id
+}
+
+# Backup Replication S3 Bucket
+
+resource "aws_s3_bucket" "graphdb_backup_replication_bucket" {
```

Comment:
We have `aws_s3_bucket_public_access_block` for the other bucket, don't we need this here too?